nxp/imx8qxp-mek/overlay.nix

```diff
@@ -0,0 +1,3 @@
+final: prev: {
+  inherit ( final.callPackage ./bsp/u-boot/imx8/imx-uboot.nix { pkgs = final; targetBoard = "imx8qxp"; }) ubootImx8 imx-firmware;
```

Comment:
`./bsp/u-boot/imx8/imx-uboot.nix` no u-boot/imx8 directory in basp. Also relative path of bsp doesn't seem ok.

Reply:
Relative paths are ok as long as they point within the repo directory structure.